src/ClientRuntime/Microsoft.Azure.Common/project.json

```diff
@@ -36,13 +36,10 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Runtime.Serialization": "",
-        "System.Threading": "",
```

Comment:
@markcowl I am not able to repro this. I am not sure what are we fixing?
2.1.3 works fine when I target an application to .NET 4.5.2 and .NET 4.6
So not sure if this fix is necessary.
here is the nuget restore output

Attempting to gather dependency information for package 'Microsoft.Azure.Common.2.1.3' with respect to project 'ConsoleApplication2', targeting '.NETFramework,Version=v4.5.2'
Gathering dependency information took 2.29 sec
Attempting to resolve dependencies for package 'Microsoft.Azure.Common.2.1.3' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Microsoft.Azure.Common.2.1.3'
Resolved actions to install package 'Microsoft.Azure.Common.2.1.3'
Retrieving package 'Hyak.Common 1.1.3' from 'nuget.org'.
Retrieving package 'Microsoft.Azure.Common 2.1.3' from 'nuget.org'.
Adding package 'Hyak.Common.1.1.3' to folder 'c:\users\localuser\documents\visual studio 2015\Projects\ConsoleApplication2\packages'
Added package 'Hyak.Common.1.1.3' to folder 'c:\users\localuser\documents\visual studio 2015\Projects\ConsoleApplication2\packages'
Added package 'Hyak.Common.1.1.3' to 'packages.config'
Successfully installed 'Hyak.Common 1.1.3' to ConsoleApplication2
Adding package 'Microsoft.Azure.Common.2.1.3' to folder 'c:\users\localuser\documents\visual studio 2015\Projects\ConsoleApplication2\packages'
Added package 'Microsoft.Azure.Common.2.1.3' to folder 'c:\users\localuser\documents\visual studio 2015\Projects\ConsoleApplication2\packages'
Added package 'Microsoft.Azure.Common.2.1.3' to 'packages.config'
Successfully installed 'Microsoft.Azure.Common 2.1.3' to ConsoleApplication2
Executing nuget actions took 841.82 ms
========== Finished ==========
Time Elapsed: 00:00:05.7560697